partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,134 @@
+    def _should_use_subprocess(self, file_size):
+        """
+        Determine whether to use a subprocess based on inflight counts and file size.
+        """
+        with PartFactoryBrep.lock:
+            if (
+                PartFactoryBrep.count_inflight_simple < PartFactoryBrep.MIN_SIMPLE_INFLIGHT
+                or file_size < PartFactoryBrep.MIN_SUBPROCESS_FILE_SIZE
+            ):
+                PartFactoryBrep.count_inflight_simple += 1
+                return False
+            else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)